app/assets/stylesheets/components/_exchange.scss

```diff
@@ -184,6 +184,31 @@ p.order-table-title {
 
 form.order-form {
     padding: 10px;
+
+    .ant-form-item {
+        margin-bottom: 0;
+    }
+
+    .ant-input {
+        text-align: right;
+    }
+
+    .ant-form-item-label {
+        text-align: left;
+        font-family: Roboto Medium, Monospaced Number, Chinese Quote,
```

Comment:
What does this do?

Reply:
Fix design of ant components for be similar to our LIMIT BUY/SELL forms as much as possible